code/HISPANIA/modules/clothing/head/jobs.dm

```diff
@@ -13,12 +13,14 @@ deberan tener una linea de codigo demas para que funcionen "hispania_icon = TRUE
 	desc = "A nice and formal hat made of silk, only for station captains. Made by D&N Corp."
 	icon_state = "caphat_dark"
 	hispania_icon = TRUE
+	species_restricted = list("exclude", "Grey", "Vox")
 
 /obj/item/clothing/head/caphat/light
 	name = "captain's formal white hat"
 	desc = "A nice and formal hat made of silk, only for station captains. Made by D&N Corp."
 	icon_state = "caphat_light"
 	hispania_icon = TRUE
+	species_restricted = list("exclude", "Grey", "Vox")
 
 //Private Sec Berets (Danaleja2005)
```

Comment:
abajo de esta linea de codigo hay algunas barets más y no estan restringidas, se ven bien en grey y vox?

Reply:
que? claro que si están restringidas

Reply:
si te refieres al private/red, ese se le hereda por ser hijo de private ( que is tiene el restricted)